RubyHW1.rb

```diff
@@ -0,0 +1,32 @@
+require "csv"
+
+CSV.open("HW1fixed.csv", "wb") do |file|
+	File.open("RubyHW1.txt", "r") do |csv|
+		csv.each do |line|
+			lin = line.split("")
+			deleteables = []
+			lin.each do |piece|
```

Comment:
Ahh this is interesting, forgot to show you something known as `.collect`. This essentially adds it into the array.

```rb
deletables = lin.collect do |piece|
  if piece != "1" && piece != "2"
     piece # by default ruby returns the last item
  end
end
```

Anything returned is collected or appended into an array.